mk/config.mk

```diff
@@ -1158,3 +1158,20 @@ $(eval $(call cfg-depends-all,CFG_WIDEVINE_HUK,CFG_DT))
 # DT node "/options/op-tee/widevine" to some specific TAs.
 CFG_WIDEVINE_PTA ?= n
 $(eval $(call cfg-depends-all,CFG_WIDEVINE_PTA,CFG_DT CFG_WIDEVINE_HUK))
+
+# Semihosting is a debugging mechanism that enables code running on an embedded
+# system (also called the target) to communicate with and use the I/O of the
+# host computer.
+CFG_SEMIHOSTING ?= n
+
+# Enable the semihosting console driver which inputs/outputs the characters
+# from/to the file or debug terminal on the semihosting host computer.
+#   - Specify CFG_SEMIHOSTING_CONSOLE_FILE="{your_log_file}" to output the
```

Comment:
nitpicking: I think it would better without the indentation:
```
# Specify CFG_SEMIHOSTING_CONSOLE_FILE="{your_log_file}" to output the
# characters to that file. Output to "optee.log" by default.
# Specify CFG_SEMIHOSTING_CONSOLE_FILE=NULL to output the characters to the
# semihosting host terminal.

Reply:
Remove two spaces in https://github.com/OP-TEE/optee_os/pull/6700/commits/35cb6b05f9ef5b50df37c0a3c543028292f82c43